eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -1,9 +1,5 @@
 resources:
   repositories:
-  - repository: azure-sdk-build-tools
```

Comment:
This is actually still used, on Line 31 below.

Reply:
Right.

Reply:
@mitchdenny do you know if there is any reason the iOS templates cannot be moved out into this repo to remove this repository reference?